Dockerfile

```diff
@@ -6,8 +6,10 @@
 
 FROM debian:bullseye-slim 
 
-ENV AEROSPIKE_VERSION 7.3.1
-ENV AEROSPIKE_SHA256 6dd56489f31286ef3124fd900627cb697d07d3661122c8519ca452724f60bb9d
```

Comment:
As Tools is not part of the Aerospike Official Image, the removal of the SHA from the Dockerfile should not raise an issue with the DockerHub folks.
But @kportertx has sent messages (IRC & Email) requesting further information on whether they will accept a Dockerfile minus the SHA.

Reply:
Ah, I did not realize it could be an issue to the DockerHub folks